server/socket/index.js

```diff
@@ -28,7 +29,14 @@ dynamicNspLobby.on("connect", (socket) => {
 
   socket.on(SocketEvents.SUBMIT_CHAT_MESSAGE, (data, callback) => {
     let isCorrect = currentLobby.evaluateGuess(data);
-    callback(isCorrect);
```

Comment:
callback can be useful for the user sending the original message to get special info if you needed, but seems like not necessary for now!